rudder-webapp/SOURCES/rudder-webapp-postinst

```diff
@@ -172,11 +172,11 @@ cd /
 
 # Restart the webapp
 echo -n "INFO: Make sure Rudder webapp is started..."
-systemctl start rudder-jetty >> ${LOG_FILE}
+systemctl restart rudder-jetty >> ${LOG_FILE}
```

Comment:
does systemd starts the service if it is not started ?

Reply:
yes 

ie with rudder-slapd

<pre>root@server:/# systemctl stop rudder-slapd
root@server:/# systemctl status rudder-slapd
● rudder-slapd.service - Rudder OpenLDAP Server Daemon
   Loaded: loaded (/usr/lib/systemd/system/rudder-slapd.service; enabled; vendor preset: enabled)
  Drop-In: /etc/systemd/system/rudder-slapd.service.d
           └─override.conf
   Active: inactive (dead) since Wed 2020-01-29 18:33:05 CET; 1s ago
  Process: 17406 ExecStart=/opt/rudder/libexec/slapd -n rudder-slapd -u rudder-slapd -f /opt/rudder/etc/openldap/slapd.conf -h ldap://0.0.0.0:389/ (code=exited, status=0/SUCCESS)
 Main PID: 17407 (code=exited, status=0/SUCCESS)

Jan 29 18:32:10 server systemd[1]: Starting Rudder OpenLDAP Server Daemon...
Jan 29 18:32:10 server rudder-slapd[17406]: @(#) $OpenLDAP: slapd 2.4.47 (Nov 22 2017 11:47:27) $
                                                    openldap
Jan 29 18:32:10 server rudder-slapd[17407]: slapd starting
Jan 29 18:32:10 server systemd[1]: Started Rudder OpenLDAP Server Daemon.
Jan 29 18:33:05 server systemd[1]: Stopping Rudder OpenLDAP Server Daemon...
Jan 29 18:33:05 server rudder-slapd[17407]: daemon: shutdown requested and initiated.
Jan 29 18:33:05 server rudder-slapd[17407]: slapd shutdown: waiting for 0 operations/tasks to finish
Jan 29 18:33:05 server rudder-slapd[17407]: slapd stopped.
Jan 29 18:33:05 server systemd[1]: Stopped Rudder OpenLDAP Server Daemon.
root@server:/# systemctl restart rudder-slapd
root@server:/# systemctl status rudder-slapd
● rudder-slapd.service - Rudder OpenLDAP Server Daemon
   Loaded: loaded (/usr/lib/systemd/system/rudder-slapd.service; enabled; vendor preset: enabled)
  Drop-In: /etc/systemd/system/rudder-slapd.service.d
           └─override.conf
   Active: active (running) since Wed 2020-01-29 18:33:12 CET; 3s ago
  Process: 17424 ExecStart=/opt/rudder/libexec/slapd -n rudder-slapd -u rudder-slapd -f /opt/rudder/etc/openldap/slapd.conf -h ldap://0.0.0.0:389/ (code=exited, status=0/SUCCESS)
 Main PID: 17425 (slapd)
    Tasks: 3 (limit: 4915)
   CGroup: /system.slice/rudder-slapd.service
           └─17425 /opt/rudder/libexec/slapd -n rudder-slapd -u rudder-slapd -f /opt/rudder/etc/openldap/slapd.conf -h ldap://0.0.0.0:389/

Jan 29 18:33:12 server systemd[1]: Starting Rudder OpenLDAP Server Daemon...
Jan 29 18:33:12 server rudder-slapd[17424]: @(#) $OpenLDAP: slapd 2.4.47 (Nov 22 2017 11:47:27) $
                                                    openldap
Jan 29 18:33:12 server rudder-slapd[17425]: slapd starting
Jan 29 18:33:12 server systemd[1]: Started Rudder OpenLDAP Server Daemon.
root@server:/# 
</pre>